File: Python/Translate.py
class Translate:
    def translate(Word):
        word =Word
        counter = len(word)
        for i in range (0,counter):
            word = word.join("o")
        print(word)

def main():
    myword = input("please enter a word you want to translate to swidish robbors word ")
    Translate.translate(myword)
if __name__=="__main__":
    main()

File: C-plus-plus/translate/Translate.cpp
#include <iostream>
#include <string>
using namespace std;
class Translate
{
    private: string myword;
    public:
        void translate(string Myword)
    {
        myword = Myword;
        for(int i=0 ; i< myword.length();i+=2)
        {
            myword.insert(i,"o");
        }
        cout<<"new translated word is "<<myword<<endl;
    }
};

int main()
{

    cout<<"please enter the word you would like to translate "<<endl;
    string word;
    cin>>word;
    Translate trs;
    trs.translate(word);
}
